smirnoff_omm/smirnoff.ipynb

```diff
@@ -4,27 +4,35 @@
    "cell_type": "markdown",
```

Comment:
<p>I need to inspect this further, but why are the values for oplsaa double what they were before?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mosdef-hub/mosdef-workflows/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
No clue ... that's concerning.